docs/queries/about.mdx

```diff
@@ -223,6 +224,15 @@ const exampleInput = screen.getByLabelText('Example')
 
 ```js
 cy.findByLabelText('Example').should('exist')
+```
+
+  </TabItem>
+  <TabItem value="svelte">
+
+```js
+import {render, screen} from '@testing-library/svelte'
+
+const exampleInput = screen.getByLabelText('Example')
```

Comment:
Should this render the component first?
I would also prefer to revert this change to keep this consistent, or all the examples in the core api should include a svelte example. If you want to, feel free to create a follow-up PR with the changes.

Reply:
I reverted this and will follow-up with a future PR to add svelte to all of the examples in the core api.